src/lib/storage/fake_device_factory.rb

```diff
@@ -242,7 +242,7 @@ def create_partition_table(parent, args)
       # @return [Fixnum]
       def str_to_ptable_type(string)
         # Allow different spelling
-        string = "msdos" if string.downcase == "ms-dos"
+        string = "msdos" if string.casecmp("ms-dos").zero?
```

Comment:
String#`casecmp` is consistent with String#`<=>` and also with (libc)  `strcmp`

The original upstream discussion for the cop (and other ones) is https://github.com/bbatsov/rubocop/issues/1677